src/apps/aave/positions.ts

```diff
@@ -111,211 +150,152 @@ const hook: PositionsHook = {
         ? `?marketName=${NETWORK_ID_TO_AAVE_MARKET_NAME[networkId]}`
         : '')
-    return reservesData.flatMap((reserveData, i) => {
-      const supplyApy = getApyFromRayApr(reserveData.liquidityRate)
-      const variableBorrowApy = getApyFromRayApr(reserveData.variableBorrowRate)
-      const stableBorrowApy = getApyFromRayApr(
-        userReserveData?.[i].stableBorrowRate || reserveData.stableBorrowRate,
-      )
-
-      // Include a token if the user has a balance
-      // or if no user data is available (when querying all positions)
-      const useAToken =
-        !userReserveData || userReserveData[i].scaledATokenBalance > 0n
-      const useVariableDebt =
-        !userReserveData || userReserveData[i].scaledVariableDebt > 0n
-      const useStableDebt =
-        !userReserveData || userReserveData[i].principalStableDebt > 0n
-
-      const userIncentives = userIncentivesData?.[i]
-      const aTokenRewardsInfo =
-        userIncentives?.aTokenIncentivesUserData.userRewardsInformation.filter(
-          (info) => info.userUnclaimedRewards > 0n,
+    return [
+      reservesData.flatMap((reserveData, i) => {
+        const supplyApy = getApyFromRayApr(reserveData.liquidityRate)
+        const variableBorrowApy = getApyFromRayApr(
+          reserveData.variableBorrowRate,
         )
-      const variableDebtRewardsInfo =
-        userIncentives?.vTokenIncentivesUserData.userRewardsInformation.filter(
-          (info) => info.userUnclaimedRewards > 0n,
-        )
-      const stableDebtRewardsInfo =
-        userIncentives?.sTokenIncentivesUserData.userRewardsInformation.filter(
-          (info) => info.userUnclaimedRewards > 0n,
+        const stableBorrowApy = getApyFromRayApr(
+          userReserveData?.[i].stableBorrowRate || reserveData.stableBorrowRate,
         )
 
-      return [
-        // AToken
-        useAToken &&
-          ({
-            type: 'app-token-definition',
-            networkId,
-            address: reserveData.aTokenAddress.toLowerCase(),
-            tokens: [
-              { address: reserveData.underlyingAsset.toLowerCase(), networkId },
-            ],
-            availableShortcutIds: ['deposit', 'withdraw'],
-            displayProps: {
-              title: reserveData.symbol,
-              description: `Supplied (APY: ${supplyApy.toFixed(2)}%)`,
-              imageUrl: AAVE_LOGO,
-            },
-            dataProps: {
-              manageUrl,
-              termsUrl: AAVE_TERMS_URL,
-              contractCreatedAt:
-                AAVE_CONTRACT_CREATED_AT[
-                  getTokenId({
-                    networkId,
-                    address: reserveData.aTokenAddress.toLowerCase(),
-                  })
-                ],
-              tvl: toSerializedDecimalNumber(
-                toDecimalNumber(totalSupplies[i], lpTokenDecimals[i]),
-              ),
-              yieldRates: [
+        // Include a token if the user has a balance
```

Comment:
It's easier to hide the whitespaces so the diff is easier to read here.

<img width="271" alt="Screenshot 2024-08-28 at 15 32 44" src="https://github.com/user-attachments/assets/d99ccdbe-49e4-49c9-a0ac-80c09fa5afa8">